### DISCUSSION TITLE
[ Warn ] Duplication of preset - Web Fonts

### DISCUSSION THREAD

**Discussion by QaahirStewart** [unocss/unocss#2536] [General]
Hey, guys. How can I get rid of this error?  I'm trying to use presetWebFonts with the Google provider and Fontshare provider. Only way i got it working is to duplicate the web font array. Ex. below.
    presetWebFonts({
      provider: "google",
      fonts: {
        Inter: "Inter",
        Poppins: "Poppins",
        Roboto: "Roboto",
        Lato: "Lato",
        Montserrat: "Montserrat",
        Pacifico: "Pacifico",
        Oswald: "Oswald",
        Raleway: "Raleway",
        Courgette: "Courgette",
        Oxygen: "Oxygen",
        Rubik: "Rubik",
        Ubuntu: "Ubuntu",
        Arvo: "Arvo",
        Caveat: "Caveat",
        Sriracha: "Sriracha",
        VT323: "VT323",
        Syncopate: "Syncopate",
        Enriqueta: "Enriqueta",
        Cabinet: "Cabinet",
      },
    }),
    presetWebFonts({
      provider: "fontshare",
      fonts: {
        Chillax: "Chillax",
        Ranade: "Ranade",
        Panchang: "Panchang",
        Zodiak: "Zodiak",
        Sentient: "Sentient",
        Supreme: "Supreme",
        Author: "Author",
        Telma: "Telma",
        Neco: "Neco",
        Alpino: "Alpino",
        Quilon: "Quilon",
        Pally: "Pally",
        Erode: "Erode",
        Nippo: "Nippo",
        Sharpie: "Sharpie",
        Bevellier: "Bevellier",
        Technor: "Technor",
        Britney: "Britney",
        Styro: "Styro",
        Hoover: "Hoover",
        Kalam: "Kalam",
        Quicksand: "Quicksand",
      },
    }),
  ],

**Comment by eminos** [comment]
I have the same issue. It seems to work but there is a warning.
```
[unocss] duplication of preset @unocss/preset-web-fonts found, there might be something wrong with your config.
```

**Reply by QaahirStewart** [reply]
yeah still works but would like the solution to clear error, no answers yet unfortunately
